concrete/single_pages/dashboard/calendar/add.php

```diff
@@ -20,8 +21,10 @@
     $enableMoreDetails = $calendar->enableMoreDetails();
     if ($enableMoreDetails == 'C') {
         $enableMoreDetails = 'create';
-    } else if ($enableMoreDetails == 'A') {
-        $enableMoreDetails = 'associate';
+    } else {
+        if ($enableMoreDetails == 'A') {
```

Comment:
`elseif` is fine to use for the record, just make sure you use `} elseif {` and not `} else if {`

Reply:
I did not worked that part, but I will keep in mind if I see it in the future 👍